src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ListenModeRenderer.scss

```
@@ -309,18 +349,12 @@
-    min-height: 80px;
-    height: 80px;
-    grid-template-columns: repeat(4, minmax(0, 1fr)) !important;
-    border-radius: 0 !important;
-    border-left: 0 !important;
-    border-right: 0 !important;
-    box-shadow: none !important;
+    --slide-player-mobile-control-count: 5 !important;
+    grid-template-columns: repeat(
+      var(--slide-player-mobile-control-count, 4),
+      minmax(0, 1fr)
+    ) !important;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)